lib/service/slack/slack-account.js

```diff
@@ -131,6 +133,8 @@ class SlackAccount extends Account {
     const {members} = await this.rtm.webClient.users.list()
     for (const m of members)
       this.users[m.id] = new SlackUser(m)
+    // Online/away status.
+    this.rtm.subscribePresence(Object.keys(this.users))
```

Comment:
Some teams can have more 20k users, subscribing to all of them would be huge. I think we should only subscribe to the ones under "Direct Messages" for now.